### PR TITLE
Develop fire life cycle discovery

### PR DESCRIPTION
**Pull request by sahmad3**
## Why are you creating this Pull Request?

Updates to Life Cycle of a Fire discovery.

*** Please don't merge this pull request to the main branch yet, issuing this pull request for preview purposes internally ***

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -8,10 +8,10 @@ APP_DESCRIPTION=Visualization, Exploration, and Data Analysis (VEDA)
 APP_CONTACT_EMAIL=email@example.org
 
 # Endpoint for the Tiler server. No trailing slash.
-API_RASTER_ENDPOINT='https://staging-raster.delta-backend.com'
+API_RASTER_ENDPOINT='https://dev-raster.delta-backend.com'
```
Comment (hanbyul-here) [inline .env:11]: Can you put these back into staging?

**Comment by hanbyul-here** [comment]
All the images are heavier than needed. Do you mind running them through  https://imageresizer.com/? (Or whichever preferred image optimizer listed in https://github.com/NASA-IMPACT/veda-config/blob/develop/docs/frontmatter/media.md#media )

**Review by hanbyul-here — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (sahmad3) [reply .env]: Just changed that!

**Comment by danielfdsilva** [comment]
@sahmad3 It would be great if the larger gifs could be provided as videos. This allows the user to decide whether or not to play them avoiding downloading such heavy files and speeding up initial load.

**Comment by danielfdsilva** [comment]
@sahmad3 Ideally the video would live on youtube or a similar site and the be included like for the "sea ice thinkness" discovery.

https://github.com/NASA-IMPACT/veda-config/blob/d2d772d0cdba2a1ad9e4b1ce16b0bf49c8ca2e3c/discoveries/sea-ice-thickness.discoveries.mdx?plain=1#L55-L70

Otherwise it can be included locally as you'd do with an image:
```jsx
<Block>
  <Figure>
     <video style={{ width: '100%', aspectRatio: "16/9" }} controls>
      <source
        src={new URL('./filename.mp4', import.meta.url).href} type="video/mp4"
      />
      Your browser does not support the video tag.
    </video>
    <Caption>
      The video caption if needed
    </Caption>
  </Figure>
</Block>
```

**Comment by sahmad3** [comment]
> 

What is the size limit for images on the discovery?

**Comment by hanbyul-here** [comment]
@sahmad3 We recommend the image be less than 500kb.

**Comment by sahmad3** [comment]
I have resized most images, hope these are acceptable now. Thanks

**Comment by danielfdsilva** [comment]
@sahmad3 The `datasets/CONUS_Nitrate_07012018.png` is still very heavy, with more than 6MB. It's pixel size can also be reduced.
If you use the tool at https://imageresizer.com/ you'll be able to resize the image (2000px on the longer side is enough), and then compress it to a much more manageable size.
A walkthrough can be found in the [veda-config docs](https://github.com/NASA-IMPACT/veda-config/blob/969cf3b50e32ba0f7fabaf73521d36e1510c2ef2/docs/frontmatter/media.md#media).

**Comment by sahmad3** [comment]
Reduced that image as well in size, thanks!

**Pull request merged**